tests/test_file.py

```diff
@@ -559,14 +560,22 @@ def test_serialize_paths():
     up_str = "s3://lamindb-ci/test-data/test.csv"
     up_upath = UPath(up_str)
 
-    _, filepath, _, _, _ = process_data("id", fp_str, None, skip_existence_check=True)
+    _, filepath, _, _, _ = process_data(
```

Comment:
This would be much more easy to read if it used `@pytest.mark.parametrize` - there are many examples for this in `test_file.py`. Could you use one of them?

Reply:
I am really not sure. I mean i don't do parametrizations for some tests because it turns into some cross lists with repeating parameters etc.
Here i would need to match different results to different parameters. Maybe it can be done, but why?

Reply:
It's ok, Sergei. I think it's easier to read but this case here is so simple that it's also OK to read without it. Keep it as is if you don't agree! :)

Reply:
Maybe I'm off here and it'd in fact be harder to read!